content/300-guides/050-database/870-using-prisma-with-mongodb.mdx

```diff
@@ -0,0 +1,242 @@
+---
+title: 'Using Prisma with MongoDB'
+metaTitle: 'Using Prisma with MongoDB'
+metaDescription: 'Guide to using Prisma with MongoDB'
+tocDepth: 2
+toc: true
+---
+
+<TopBlock>
+
+This guide discusses the concepts behind using Prisma and MongoDB, explains the commonalities and differences between MongoDB and other database providers, and leads you through the process for configuring your application to integrate with MongoDB.
+
+</TopBlock>
+
+## What is MongoDB?
+
+MongoDB is a NoSQL database that stores data in a JSON-like document format.
```

Comment:
That JSON-like format is called BSON.

Reply:
Yep, will most likely include that when this placeholder gets rewritten